s4/str_brackets.rb

```diff
@@ -0,0 +1,35 @@
+# brackets for extract characters
+sbracket = "Esta es una oración."
+pos = 17
+p sbracket
+puts "Hay una #{sbracket[pos]} en la posición #{pos}";
+print "[17] = "; p sbracket[17]
```

Comment:
What happens if the index is greater than the length of the string?

Reply:
I’d return nil, just like I tested it on line 8.

> `print "[100] = "; p sbracket[100]`